### ISSUE TITLE
Google summer of code participacion 

### ISSUE THREAD

**Issue by shlyakpavel** [lxqt/lxqt#1651]
Hello. I guess some of you already know about this project.
Participation there could bring here new talanted developers. Possible tasks:
- Write new lightweight terminal widget (not a fork of Konsole, clean and fresh)
- Rework screengrab uploader
And a lot of other ideas already mentioned in the issues.
So was there an attempt to attend it? It would be really nice to hear up to date information on it.
Have a nice day!

**Comment by agaida** [comment]
* screengrab uploader is history - a general upload module/library/something else would be nice
* terminalwidget from the scratch? erm - not really imho

https://github.com/lxqt/lxqt/issues/1651 - was a task from IRC, one of the things where we really fall short

**Comment by tsujan** [comment]
> terminalwidget from the scratch? erm - not really imho

I also see no reason whatsoever to rewrite it. We can make it better, as we have done (thanks to @yan12125).

**Comment by the-loudspeaker** [comment]
#1196 ?

**Comment by tsujan** [comment]
> #1196 ?

I still think that a "favorite apps" section is a duplicate of Quick Launch, regardless of what other DEs do. I also ask why making lxqt-panel heavier by adding a duplication? So, I'm not interested. But I know people may love their habits...

**Comment by the-loudspeaker** [comment]
Understandable.

**Comment by the-loudspeaker** [comment]
Other ideas @tsujan ?

**Comment by tsujan** [comment]
> Other ideas @tsujan ?

About?

Please note that other LXQt developers may have different opinions. That may be the reason why https://github.com/lxqt/lxqt/issues/1196 is still open. Moreover, contributors may have good ideas and make PRs.

**Comment by agaida** [comment]
[Feature Request] Add support for 3rd level characters/AltGr key #1110

^^^^ this one will be important - not for me, but for users who use non-latin letters

**Comment by wxl** [comment]
I agree with #1110 and emphasize that it's also useful for anyone to configure any keyboard options, including stuff like:
 * compose key
 * control key behavior
 * caps lock behavior
 * alt/win key behavior

**Comment by wxl** [comment]
BTW see the [forum](https://forum.lxqt.org/t/google-summer-of-code-2020/1067/2) on this subject.

Also here's some things that would be appropriate, I think:

 * easy
   * fix terms lxqt/lxqt-globalkeys#127
   * desktop entry metadata fix on lxqt-config-input lxqt/lxqt-config#524
   * informative tooltips lxqt/lxqt-config#494
   * LXQt icon in menu lxqt/lxqt-about#128
 * docs
   * help! lxqt/pcmanfm-qt#550
   * moar help lxqt/qterminal#502
 * maybe too hard?
   * allow movement to different monitors lxqt/lxqt-panel#965
   * fix `menu` URI interpretation lxqt/pcmanfm-qt#329
 * normal stuff
   * folders in quick launch lxqt/lxqt-panel#912
   * network speed in panel widget lxqt/lxqt-panel#911
   * disable "Create New" when impossible lxqt/pcmanfm-qt#669
   * symlink editing lxqt/pcmanfm-qt#319
   * two apply buttons? lxqt/lxqt-config#577
   * don't close with multiple tabs lxqt/qterminal#496
   * rename tab should show current name lxqt/qterminal#499

Still need to dig some more through bugs in the lxqt repo. There's a lot.

**Comment by stefonarch** [comment]
What about more samba integration? "Share this folder" + gui for smb.conf?
Maybe open an issue about this?

**Comment by tsujan** [comment]
> What about more samba integration?

I think it requires checking smb at startup. Have you seen Dolphin? Its startup is delayed because of that. I'm not saying that the same undesirable delay is inevitable with libfm-qt but I think SMB is too specific to be included in the code.

What I said above is not my final decision though. I should try to do it first (when I find the time). So, IMO, it deserves a feature request. Would you make one?

**Comment by tsujan** [comment]
@wxl 

> disable "Create New" when impossible lxqt/pcmanfm-qt#669

That's a piece of cake. My mistake was that I waited for a user who wanted to do it.

**Comment by stefonarch** [comment]
> IMO, it deserves a feature request. Would you make one

https://github.com/lxqt/lxqt/issues/1763
